Responsive Images/image.css

```diff
@@ -0,0 +1,80 @@
+:root {
+  --gray: #3e4152;
+  --blue: #084298;
+  --text-color: #007fff;
+  --white-bg: #fafafa;
+}
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 2rem;
+  color: var(--gray);
+}
+h1 {
+  font-size: 3rem;
+}
+
+h1, h2, p{
+  margin: 0.625rem 0;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 2rem;
+}
+p {
+  font-size: 1.2rem;
+  padding: 0.8rem;
+}
```

Comment:
If you are this same thing in multiple components then simply put this in your utility CSS.